include/message_struct.h

```diff
@@ -0,0 +1,12 @@
+#ifndef _MESSAGE_DATA_STATIC_H_
+#define _MESSAGE_DATA_STATIC_H_
+#include "PR/ultratypes.h"
+
+struct MessageTableEntry{
```

Comment:
Btw, OoT has this struct too, on a header named `message_data_static.h`. I think it would be better to name this header like that to keep consistency between both games.

Reply:
As the header guard suggest, I forgot to actually change the filename, done in acd12b2